R/genomeSpec.R

```diff
@@ -57,29 +37,20 @@ common.always.available.stock.genomes <- function() {
 #' @export
 #'
 currently.supported.stock.genomes <- function(test = FALSE) {
```

Comment:
@karollayna, you are right. Changing the getter to sth with the verb in the name (e.g. `get_css_genomes`) makes sense.

Reply:
ok